soc/nordic/nrf52/Kconfig

```diff
@@ -31,12 +31,14 @@ config SOC_NRF52840
 if SOC_SERIES_NRF52X
 
 config SOC_DCDC_NRF52X
-	bool
+	bool "DC/DC mode"
+	depends on $(dt_nodelabel_bool_prop,power,dcdc-components)
 	help
 	  Enable nRF52 series System on Chip DC/DC converter.
 
 config SOC_DCDC_NRF52X_HV
-	bool
+	bool "High Voltage DCDC converter"
+	depends on $(dt_nodelabel_bool_prop,power,hv-dcdc-components)
```

Comment:
ok to have a Kconfig setting which resembles the dts setting, but please use the status property from devicetree, that is `status = "disabled"` / `status = "okay"` instead of controlling the Kconfig based on whether the node is defined or not.